masonry/src/testing/harness.rs

```diff
@@ -570,7 +582,7 @@ impl TestHarness {
             // Remove '<test_name>.new.png' file if it exists
             let _ = std::fs::remove_file(&new_path);
             new_image.save(&new_path).unwrap();
-            panic!("No reference file");
+            panic!("Snapshot test '{test_name}' failed: No reference file");
```

Comment:
Great. I am planning on making these snapshot tests *much* more user-friendly, but this is a good start.